README.md

```diff
@@ -1 +1,94 @@
-# buget-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Budget-app
+> Money Wallet is a mobile web application that allows you to manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+----
+_[Here](https://drive.google.com/file/d/1gzTESqGQUBQ-7ebR3fyBBLy7vVLLDWYq/view?usp=sharing) is the link to the project presentation video_
+
+_[Here](https://powerful-stream-89137.herokuapp.com/) is the link to the live demo_
+
+---
+  ![Screenshot from 2022-09-09 01-25-18](unavailable)
```

Comment:
- [Optional] The link to this screenshot is broken. I'll kindly advise you to fix it. This way, your README file would become more descriptive.